src/layout/Header/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import "./style.css";
+
+export default () => {
+  const username = useSelector((store) => store.username);
```

Comment:
creative, never seen store used like this👀

Reply:
lol... its original work...